CIP-0110/README.md

```diff
@@ -0,0 +1,87 @@
+---
+CIP: 110
+Title: Plutus v1 Script References
+Status: Proposed
+Category: Plutus
+Authors:
+    - Pi Lanningham <pi@sundaeswap.finance>
+Implementors: []
```

Comment:
```suggestion
Implementors:
    - Alexey Kuleshevich <alexey.kuleshevich@iohk.io>
```
Would be able to cite you as the implementor of this @lehins?
Similar to how we cite Michael PJ for Plutus CIPs, [example](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0101/README.md?plain=1#L10).